test/browser_tests/spec_suites/careers-application-process.js

```diff
@@ -52,4 +52,31 @@ describe( 'The Application Process Page', function() {
     expect( page.relatedLinksSection.isPresent() ).toBe( true );
   } );
 
+  describe( '(Video Player)', function() {
+    it( 'should be present', function() {
+      expect( page.videoPlayer.isPresent() ).toBe( true );
+    } );
+
+    it( 'display show an image when the page loads', function() {
+      expect( page.videoPlayerImageContainer.isDisplayed() ).toBe( true );
+      expect( page.videoPlayerVideoContainer.isDisplayed() ).toBe( false );
+    } );
+
+    it( 'should show the video when the play button is clicked', function() {
+      page.videoPlayerPlayButton.click();
+      expect( page.videoPlayerVideoContainer.isDisplayed() ).toBe( true );
+      expect( page.videoPlayerImageContainer.isDisplayed() ).toBe( false );
+    } );
+
+    it( 'should attach an iframe when the play button is clicked', function() {
+      expect( page.getVideoPlayerIframe().isPresent() ).toBe( true );
```

Comment:
It would be better not to have the tests depend on each other or on execution order. Like if someone needed to `xit` L:65, the test on L:71 would start not responding as expected. 
This could be solved with a describe block like:

```
describe( 'when play button is clicked', function() {
      beforeEach( function() {
        page.videoPlayerPlayButton.click();
      } );

     it( 'should show the video', function() {
       expect( page.videoPlayerVideoContainer.isDisplayed() ).toBe( true );
       expect( page.videoPlayerImageContainer.isDisplayed() ).toBe( false );
     } );

     it( 'should attach an iframe', function() {
       expect( page.getVideoPlayerIframe().isPresent() ).toBe( true );
     }
} );
```